react-frontend/src/pages/Home/Home.jsx

```diff
@@ -1,75 +1,58 @@
 import { useGetLatestCo2Value } from "../../utils/api/useGetLatestCo2Value";
-import { useGetLatestTemperatureValue } from "../../utils/api/useGetLatestTemperatureValue";
-import { useGetLatestHumidityValue } from "../../utils/api/useGetLatestHumidityValue";
-import './home.css';
+import "./home.css";
 import Value from "../../components/Value/Value";
+import Card from "../../components/Card/Card";
+import { useGetDailyAverageCo2Value } from "../../utils/api/useGetDailyAverageCo2Value";
 
 const AVG_OUTDOOR_CO2_LEVEL_PPM = 400;
 const AVG_ROOM_CO2_LEVEL_PPM = 1000;
 
-const COLD_INDOOR_TEMPERATURE_DEGREES = 18;
-const LOW_INDOOR_TEMPERATURE_DEGREES = 20;
-const HIGH_INDOOR_TEMPERATURE_DEGREES = 23;
-const WARM_INDOOR_TEMPERATURE_DEGREES = 25;
-
-const VERY_LOW_INDOOR_HUMIDITY_PERCENT = 20;
-const LOW_INDOOR_HUMIDITY_PERCENT = 30;
-const HIGH_INDOOR_HUMIDITY_PERCENT = 50;
-const VERY_HIGH_INDOOR_HUMIDITY_PERCENT = 60;
-
 const calculateCarbonDioxideThreshold = (co2) => {
   if (co2 < AVG_OUTDOOR_CO2_LEVEL_PPM) {
-    return 'good'
+    return "good";
   }
   if (co2 < AVG_ROOM_CO2_LEVEL_PPM) {
-    return 'medium'
-  }
-  return 'bad'
-}
-
-const calculateTemperatureThreshold = (temperature) => {
-  if (temperature >= COLD_INDOOR_TEMPERATURE_DEGREES && temperature <= LOW_INDOOR_TEMPERATURE_DEGREES) {
-    return 'medium'
-  }
-  if (temperature >= HIGH_INDOOR_TEMPERATURE_DEGREES && temperature <= WARM_INDOOR_TEMPERATURE_DEGREES) {
-    return 'medium'
-  }
-  if (temperature < COLD_INDOOR_TEMPERATURE_DEGREES) {
-    return 'bad'
-  }
-  if (temperature > WARM_INDOOR_TEMPERATURE_DEGREES) {
-    return 'bad'
-  }
-  return 'good'
-}
-
-const calculateHumidityThreshold = (humidity) => {
-  if (humidity > VERY_LOW_INDOOR_HUMIDITY_PERCENT && humidity <= LOW_INDOOR_HUMIDITY_PERCENT) {
-    return 'medium'
-  }
-  if (humidity > HIGH_INDOOR_HUMIDITY_PERCENT && humidity <= VERY_HIGH_INDOOR_HUMIDITY_PERCENT) {
-    return 'medium'
-  }
-  if (humidity < VERY_LOW_INDOOR_HUMIDITY_PERCENT) {
-    return 'bad'
-  }
-  if (humidity > VERY_HIGH_INDOOR_HUMIDITY_PERCENT) {
-    return 'bad'
+    return "medium";
   }
-  return 'good'
-}
+  return "bad";
+};
 
 const Home = () => {
-  const { state: tState, data: tData } = useGetLatestTemperatureValue()
-  const { state: cState, data: cData } = useGetLatestCo2Value()
-  const { state: hState, data: hData } = useGetLatestHumidityValue()
+  const { state: cState, data: cData } = useGetLatestCo2Value();
```

Comment:
the variable names `cState` and `cData` made more sense when we had the other sensor data. What do you think about using the names `liveState` and `liveData`. Thoughts?

Reply:
I agree, it makes more sense since we only have co2 data now. Will update this.

Reply:
This is fixed now!